lib/couchdb.js

```diff
@@ -15,11 +15,15 @@ var url = require('url');
 var util = require('util');
 var _ = require('lodash');
 
+const DEFAULT_MODEL_VIEW = 'loopback__model__name';
+const DEFAULT_MODEL_PREFIX = 'LBModel';
+const DEFAULT_PROPERTY_PREFIX = 'LBIndex';
```

Comment:
+1, that's a very good point that I wasn't considering. I think for now we should leave it as-is. We can extend that functionality in a non-breaking fashion later.